src/lib/libdylink.js

```diff
@@ -668,6 +669,7 @@ var LibraryDylink = {
 #endif
         wasmTable.grow({{{ toIndexType('metadata.tableSize') }}});
       }
+      usedTableLength = tableBase + metadata.tableSize;
```

Comment:
I think this line should be moved up inside the above `if (metadata.tableSize)` check, no?

Reply:
I think this makes sense.

Reply:
Changed